plan/logical_test.go

```diff
@@ -210,14 +215,15 @@ func (PushFilterThroughMapRule) Pattern() plan.Pattern {
 			plan.Any()))
 }
 
-func (PushFilterThroughMapRule) Rewrite(pn plan.PlanNode) (plan.PlanNode, bool) {
+func (PushFilterThroughMapRule) Rewrite(pn plan.PlanNode) (plan.PlanNode, bool, error) {
```

Comment:
As we add more rewrite rules this file seems a little redundant to me. Maybe we don't need it anymore when we have these corresponding rules in filter.go and map.go?

Reply:
Yes, I agree this test will become redundant as it is now.